front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -77,8 +81,14 @@ const SimulationResults = ({
     pathProperties
   );
 
-  // Compute path items coordinates in order to place them on the map
+  const [projectPathTrainResult, setProjectPathTrainResult] = useState<TrainSpaceTimeData[]>([]);
+  useEffect(() => {
```

Comment:
Improves readability :)

```suggestion
  const [projectPathTrainResult, setProjectPathTrainResult] = useState<TrainSpaceTimeData[]>([]);
  
  useEffect(() => {
```